### PR TITLE
Support for user deletion

### PR DESCRIPTION
**Pull request by matyis1**
#65

### CODE REVIEW

File: db/users.js [hunk 1]
```diff
@@ -102,4 +102,33 @@ const updateName = async (email, first_name, last_name) => {
   )[0];
 };
 
-module.exports = { createUser, getUser, updateSettings, updateName };
+/**
+ *
+ * Attempts to delete a user and their card entries in the database.
+ *
+ * @param {integer} id The email of the user to update.
```
Comment (matyis1) [inline db/users.js:109]: *email

**Comment by paytonshaltis** [comment]
Looks great @matyis1!

I just went in to connect it to the delete button, and everything seems to be working! Deleting a Goon Card account seems to successfully delete all of the user's associated accounts and all of the card entries that they have generated.

**Comment by paytonshaltis** [comment]
@sderacy @lelekaz @thadakn1 could some of you three try out the following?

1) Generate a card with at least 2 accounts.
2) Open the card in another tab (or on your phone).
3) Delete one of the accounts in that card, save changes. Refresh the window or phone looking at the card. The account that was deleted should be gone, but the other should remain.
4) Now delete the entire Goon Card account. Refresh the window or phone looking at the card and you should be on the 404 page.

Once someone else confirms that this is working as intended, we can approve and merge this PR!

**Review by thadakn1 — APPROVED** [review]
works great!!

**Pull request merged**